packages/gatsby-theme-carbon/src/components/PageHeader/PageHeader.module.scss

```diff
@@ -1,28 +1,19 @@
 @import '~carbon-components/scss/globals/scss/layout';
 
 .page-header {
-  background: $carbon--black-100;
+  background: $ui-05;
```

Comment:
Is there a reason we changed this from black to gray-100?

Reply:
Whoops, was trying to switch to use theme tokens and misread this.